src/lbaf/Execution/lbsRuntime.py

```diff
@@ -49,10 +49,10 @@ def __init__(self, phases: dict, work_model: dict, algorithm: dict, arrangements
             self.__logger,
             rank_qoi,
             object_qoi)
-        if not self.__algorithm:
-            self.__logger.error(
-                f"Could not instantiate an algorithm of type {self.__algorithm}")
-            raise SystemExit(1)
+        # if not self.__algorithm:
```

Comment:
@ppebay This block will never be reached; calling lbsAlgorithmBase.factory() with an invalid algorithm name will already throw an error:
```python
        # Ensure that algorithm name is valid
        algorithm = locals()[algorithm_name + "Algorithm"]
        return algorithm(work_model, parameters, logger, rank_qoi, object_qoi)
        try:
            # Instantiate and return object
            algorithm = locals()[algorithm_name + "Algorithm"]
            return algorithm(work_model, parameters, logger, rank_qoi, object_qoi)
        except Exception as e:
            # Otherwise, error out
            logger.error(f"Could not create an algorithm with name {algorithm_name}")
            raise SystemExit(1) from e
```

Therefore we can safely remove this `if` block, since the error is handled within the base class.